### DISCUSSION TITLE
Unable to bed mesh

### DISCUSSION THREAD

**Discussion by martinn1997** [dz0ny/klipper-prusa-mk3s#36] [Q&A] [answered]
I have just installed klipper, setted up everything and I'm not able to finish bed mesh... I am getting "Probe triggered prior to movement" error randomly while printer is probing bed. I have properly setted z offset (1.670 value from original Prusa Marlin) and I have no idea what to do now. All probe and bed_mesh settings are same as in sample config in git. I have BearExxa if this can make any difference...

Help please :)

**Comment by agravelot** [comment, marked as answer]
Here two solution that fixed this issue on my side, second one is better imo:
- increase `horizontal_move_z` above 5, 7 was good for me.
- Try this magnets faulty regions:
```
faulty_region_1_min: 100.583, 187.376
faulty_region_1_max: 122.083, 229.376
faulty_region_2_min: 125.672, -18.124
faulty_region_2_max: 147.172, 23.876
faulty_region_3_min: 192.261, 5.126
faulty_region_3_max: 234.261, 26.626
faulty_region_4_min: 114.422, 39.876
faulty_region_4_max: 135.922, 81.876
faulty_region_5_min: 21.422, 87.126
faulty_region_5_max: 42.922, 129.126
faulty_region_6_min: 54.172, 97.376
faulty_region_6_max: 96.172, 118.876
faulty_region_7_min: 154.172, 97.376
faulty_region_7_max: 196.172, 118.876
faulty_region_8_min: 205.136, 87.126
faulty_region_8_max: 226.636, 129.126
faulty_region_9_min: 114.422, 134.376
faulty_region_9_max: 135.922, 176.376
faulty_region_10_min: 176.177, 191.394
faulty_region_10_max: 218.177, 212.894
```

**Reply by martinn1997** [reply]
These faulty regions are already in the config. But I've changed horizontal_move_z to 5 and works perfectly! Thanks!
